packages/bluefish.rb

```diff
@@ -47,7 +47,7 @@ def self.postinstall
     puts
     puts 'To complete the installation, execute the following:'.lightblue
     puts "update-mime-database #{CREW_PREFIX}/share/mime".lightblue
-    puts "echo 'alias bluefish=\"WAYLAND_DISPLAY=wayland-0 DISPLAY=\'\' GDK_BACKEND=wayland bluefish\"' >> ~/.bashrc".lightblue
+    puts "echo 'alias bluefish=\"WAYLAND_DISPLAY=wayland-0 DISPLAY=\'\' GDK_BACKEND=wayland bluefish\"' >> #{CREW_PREFIX}/etc/env.d/profile".lightblue
```

Comment:
Actually, why don't we remove this line from the postinstall section and add it to the package instead to `#{CREW_DEST_PREFIX}/etc/env.d/bluefish`.  That way we don't have to append the main profile which could potentially end up with duplicate lines and the lines are removed when the package is removed.

Reply:
It won't hurt to actually run the `update-mime-database` command without prompting too.

Reply:
Is it required to move the writing to `#{CREW_DEST_PREFIX}/etc/env.d/bluefish` out of postinstall? It's still in postinstall in fish.

Reply:
Yes, it can go in the `def self.build` section instead.